wait/member.go

```diff
@@ -47,6 +50,101 @@ func (a *MemberAwaitility) WaitForUserAccount(name string, expSpec toolchainv1al
 	})
 }
 
+func (a *MemberAwaitility) WaitForNSTmplSet(name string) error {
```

Comment:
I just realized that it will probably not work in your case... but you can change the `WaitForNSTmplSetWithConditions` to:
```go
func (a *MemberAwaitility) WaitForNSTmplSet(name string, waitCond ...toolchainv1alpha1.Condition) error {
	return wait.Poll(RetryInterval, Timeout, func() (done bool, err error) {
		nsTmplSet := &toolchainv1alpha1.NSTemplateSet{}
		if err := a.Client.Get(context.TODO(), types.NamespacedName{Name: name, Namespace: a.Ns}, nsTmplSet); err != nil {
			if errors.IsNotFound(err) {
				a.T.Logf("waiting for availability of NSTemplateSet '%s'", name)
				return false, nil
			}
			return false, err
		}
		if len(waitCond) == 0 || test.ConditionsMatch(nsTmplSet.Status.Conditions, waitCond...) {
			a.T.Logf("found NSTemplateSet '%s'", name)
			return true, nil
		}
		a.T.Logf("waiting for conditions match for NSTemplateSet '%s'", name)
		return false, nil
	})
}
```

Reply:
Done https://github.com/codeready-toolchain/toolchain-e2e/pull/5/commits/1e2ee0b14b05514a093aed21ed0609b14c19cd04